src/js/actions/layereffects.js

```diff
@@ -333,6 +333,22 @@ define(function (require, exports) {
     };
     setShadowSpread.reads = [locks.PS_DOC, locks.JS_DOC];
     setShadowSpread.writes = [locks.PS_DOC, locks.JS_DOC];
+    
+    /**
+     * Set the Drop Shadow Blend Mode value for all selected layers
+     *
+     * @param {Document} document
+     * @param {Immutable.Iterable.<Layer>} layers list of layers to update
+     * @param {number} shadowIndex index of the Drop Shadow within the layer(s)
+     * @param {string} blendMode  blendMode name
+     * @return {Promise}
+     */
+    var setShadowBlendMode = function (document, layers, shadowIndex, blendMode, type) {
+        return _upsertShadowProperties.call(
+           this, document, layers, shadowIndex, { blendMode: blendMode }, null, type);
+    };
+    setShadowBlendMode.reads = [locks.PS_DOC, locks.JS_DOC];
+    setShadowBlendMode.writes = [locks.PS_DOC, locks.JS_DOC];
```

Comment:
one school of thought says that you oughtn't include read locks for which you are specifying the associated write lock.  If this goes against the general pattern elsewhere in this file, then disregard